### DISCUSSION TITLE
Generating OG doesn't work for specific image(pages router)

### DISCUSSION THREAD

**Discussion by thebergamo** [vercel/community#3432] [Feature Question]
### Question

Hi there!

I'm using pages router to deploy my blog at Vercel and it seems to only for a specific image (on one blog post) it cannot generate the OG image, but at the same time it basically returns `200 OK` and I can't see any further information on logs at Vercel.

Locally it works, but after deploy there's nothing I can do do improve find what exactly is happening.

Basically I follow this pattern:

API -> `/api/og?title=Title Goes here&subtitle=Subtitle is amazing&image=/static/img/posts/post_folder/cover.jpg`

And I basically reference the image in the headers for it, I can see it for any other images, but not for one specifically. 

Initially I thought it could be because of the size, but even other images with similar size it does work, and even after reducing image size still broken.

Is there any other ways that I can debug what is happening?

Thank you very much in advance :)

**Comment by amyegan** [comment]
Hey @thebergamo. I wasn't able to replicate the problem in my test app. I think this is being caused by something more complex in your code. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give more context.

**Reply by thebergamo** [reply]
Hi @amyegan thank you for your reply.

https://github.com/thebergamo/www/pull/2

I'm implementing it in this branch, if you would have access to more detailed execution logs I can also share my Vercel app deployment to check.
The image that seems to be causing issues is https://github.com/thebergamo/www/blob/main/public/static/img/posts/mfe/mfe-cover.jpg
